.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,10 @@
+Code of Conduct
+---------------
+
+At webpack and all webpack/webpack-contrib repositories we follow the
+[JSFoundation Code of Conduct][1]. Please adhere to the guidelines there and
+feel free to report any violation of them to the @webpack/core-team or
+@webpack/documentation-team.
```

Comment:
@bebraw just fixed the link checking issue -- any word on what contact info we can add for reporting?